package.json

```diff
@@ -27,6 +27,8 @@
     "doc:typedoc": "typedoc --out doc/typedoc/ src/",
     "generate": "npm-run-all clean compile test:unit doc",
     "build": "npm-run-all clean compile",
+    "watch": "npm-run-all test watch:compile",
+    "watch:compile": "node script/webpack-client.js",
```

Comment:
Not sure about a good name for this one:
- could be just `dev` as we are developing using this target (that uses watch mode)
- however, when having node-only code you would use unit tests to develop, making this task a browser-only option
- it could be something in the `compile:` range, but since all compiles are used for 'building', it might be misplaced

So I would go for the simplest option for now until we figure out something better; `dev`